build.sbt

```diff
@@ -44,14 +40,14 @@ val jlineVersion = "2.14.3"
 
 val printDependencyClasspath = taskKey[Unit]("Prints location of the dependencies")
 
-val sharedSettings = assemblySettings ++ scalariformSettings ++ Seq(
+val sharedSettings = com.typesafe.sbt.SbtScalariform.scalariformSettings ++ Seq(
```

Comment:
can we just remove scalariform? scalafmt is the winner here.

Reply:
Should we follow up with this separately?